TestApi/TestApi.Common/Configuration/ServicesConfiguration.cs

```diff
@@ -8,5 +8,7 @@ public class ServicesConfiguration
         public string UserApiResourceId { get; set; }
         public string VideoApiResourceId { get; set; }
         public string VideoApiUrl { get; set; }
+
+        public string TestApiUrl { get; set; }
     }
```

Comment:
You don't need to add this here.

Reply:
This is needed to set Zap scan url

Reply:
Yes, but you don't need to duplicate the property when it already exists in the base class